### PR TITLE
Create fx forward read/write examples

### PR DESCRIPTION
**Pull request by ryangarvey**

### CODE REVIEW

File: examples/lusid/fx-forwards/create-fx-forward.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-- ===============================================================
+-- Description:
+-- In this query, we add an Fx Forward holding in to LUSID.
+-- ===============================================================
+
+-- Defining scope variable
+@@instrumentScope = select 'luminesce-examples';
```
Comment (DavidWassellFinbourne) [inline examples/lusid/fx-forwards/create-fx-forward.sql:7]: I think Stephen wanted a space between these comments and the code section, could you add it and for the below sections too?

File: examples/lusid/fx-forwards/create-fx-forward.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+-- Load forward into table
+@fx_data = 
+use Drive.Csv
+--file=/luminesce-examples/fx_forward.csv
+enduse;
+
+-- Define informtion to upsert from file
+@forward_to_upload =
```
Comment (DavidWassellFinbourne) [inline examples/lusid/fx-forwards/create-fx-forward.sql:16]: Could you rename this to @fx_forward_instrument

File: examples/lusid/fx-forwards/read-fx-forward.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+-- ===============================================================
+-- Description:
+-- In this query, we read an Fx Forward holding from LUSID.
+-- ===============================================================
+
+-- Read Fx Forward from LUSID based on display name.
+select * from lusid.Instrument.FxForward
+where DisplayName = 'GBPUSD 01 Feb 21';
```
Comment (DavidWassellFinbourne) [inline examples/lusid/fx-forwards/read-fx-forward.sql:8]: Could the where statement use the Figi as well?

**Review by DavidWassellFinbourne — COMMENTED** [review]
Looks good other than a few minor comments.

File: examples/lusid/fx-forwards/create-fx-forward.sql [hunk 4]
```diff
@@ -0,0 +1,30 @@
+-- ===============================================================
+-- Description:
```
Comment (DavidWassellFinbourne) [inline examples/lusid/fx-forwards/create-fx-forward.sql:2]: Could the description match that of the upload equity example with the bullet point breakdown? It is a bit overkill but makes it very clear what is happening.

**Review by DavidWassellFinbourne — COMMENTED** [review]
Just 1 additional comment.

File: examples/lusid/fx-forwards/create-fx-forward.sql (hunk 1 above)
Comment (ryangarvey) [reply examples/lusid/fx-forwards/create-fx-forward.sql:7]: Newline spacing added to both files

File: examples/lusid/fx-forwards/create-fx-forward.sql (hunk 2 above)
Comment (ryangarvey) [reply examples/lusid/fx-forwards/create-fx-forward.sql:16]: Renamed table

File: examples/lusid/fx-forwards/create-fx-forward.sql (hunk 4 above)
Comment (ryangarvey) [reply examples/lusid/fx-forwards/create-fx-forward.sql:2]: Formatted matched

File: examples/lusid/fx-forwards/create-fx-forward.sql [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+-- 1. In this query, we run an ETL process on a Fx Forward.
```
Comment (DavidWassellFinbourne) [inline examples/lusid/fx-forwards/create-fx-forward.sql:3]: Very (very) minor thing, as only 1, can it say 'an fx forward...'

**Review by DavidWassellFinbourne — COMMENTED** [review]
All looks good other than 1 incredibly minor thing (sorry).

File: examples/lusid/fx-forwards/create-fx-forward.sql (hunk 5 above)
Comment (ryangarvey) [reply examples/lusid/fx-forwards/create-fx-forward.sql:3]: My English skills letting me down....

**Review by DavidWassellFinbourne — APPROVED** [review]
All looks good.

**Review by DavidWassellFinbourne — APPROVED** [review]
Looks good!

**Review by DavidWassellFinbourne — APPROVED** [review]
Looks good

**Review by DavidWassellFinbourne — APPROVED** [review]
All good

**Pull request merged**